problems/kattis/quadrant/java/jasonBengtson/Quadrant.md

```diff
@@ -0,0 +1,45 @@
+﻿
+
+
+> Written with [StackEdit](https://stackedit.io/).
+> # Quadrant
+
+This is a solution to the (Problem ID: quadrant) on Kattis, using Java.
+The code will take an input for an x value and a y value and return the number of the quadrant the point will land on.
+
+
+## Installation
+
+If the user does not have java installed there are simple instructions of both how to install and compile java at 
+
+you can install one of the following ides if not already installed
+
+- Intellij 
+     https://www.jetbrains.com/idea/download/#section=windows
+
+- eclipse
+ https://www.eclipse.org/downloads/packages/release/kepler/sr1/eclipse-ide-java-developers
+
+
+or command line installation
+
+https://medium.com/@andrii.bibik.dev/tutorial-on-how-to-install-java-some-basics-of-command-prompt-java-file-compilation-and-run-on-740bb00e6c90
+
```

Comment:
maybe include the pertinent points in the readme

Reply:
Roger that